.cspell.json

```diff
@@ -677,7 +677,8 @@
     "subresource",
     "htmlonly",
     "apidevtools",
-    "contenteditable"
+    "contenteditable",
+    "conjuction"
```

Comment:
This is a typo, it should be `conjunction` - let's fix up the description typo and remove this line.